src/components/board/Stack.vue

```diff
@@ -24,6 +24,14 @@
 <template>
 	<div class="stack">
 		<div v-click-outside="stopCardCreation" class="stack__header" :class="{'stack__header--add': showAddCard }">
+			<Actions v-if="canEdit">
+				<ActionButton
+					id="drag_indicator"
```

Comment:
An id should only be used once, here it would be used multiple times (once for each list)